.github/workflows/main.yml

```
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        java: [ '8', '11', '16', '17-ea']
+        java: [ '8', '11', '16', '17-ea', '19']
```

LTS versions and the latest

```suggestion
        java: [ '8', '11', '17', '19']
```